### DISCUSSION TITLE
Octernship FAQ

### DISCUSSION THREAD

**Discussion by Tyagiquamar** [community/community#52185] [GitHub Education]
I submitted the octernship task before the deadline, but now there is a new assignment in textifyAI octernship. But there is no new readme file or any other thing.
![Screenshot (7)](https://user-images.githubusercontent.com/104281681/230443348-4014279d-830b-4440-83c5-f2e15290c411.png)

**Comment by samus-aran** [comment]
Hi @Tyagiquamar 

We would recommend students working on Octernship projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All project’s may have different preferred methods of communication but here are a few places to check where best to raise inquiry:


- Octernships Project’s README - This may include instructions on how and where to raise an inquiry.
- Octernships Project’s Discussions - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- Octernships Project’s Issues - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍
